apps/editing-toolkit/editing-toolkit-plugin/wpcom-universal-themes/index.php

```diff
@@ -26,8 +26,16 @@ function is_core_fse_active() {
 		return false;
 	}
 
-	// Now we just check for our own option.
-	return (bool) get_option( ACTIVATE_FSE_OPTION_NAME );
+	// For universal themes, we check for our own option.
+	if ( 'pub/blockase' === get_option( 'template' ) ) {
```

Comment:
Will the `pub/` prefix have to be removed for Atomic sites?

Reply:
Good question. I'm not sure, but it's 100% worthwhile to verify.

Reply:
Looking at phpMyAdmin on an FSE eligible atomic site, you're right. The option is `blockbase`, and not `pub/blockbase`.